app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
```

Comment:
- [ ] Kindly center your sign-in and sign-up content to the center of the page, currently, it is aligned to the left, please align to the center so your design matches the wireframe. This was requested

![image](https://github.com/firstpj/Ruby-blog/assets/81932478/5806fd79-70ba-4921-9627-3149313038be)